backend/controllers/UserController.ts

```diff
@@ -0,0 +1,37 @@
+import { Response, Request } from "express";
+import User from "../models/UserModel";
+
+const UserController = {
+  async getUserData(req: Request, res: Response) {
+    try {
+      const userToken = req.params.userToken;
```

Comment:
We will get token from the header (Bearer Token)
const token = req.header('Authorization')?.split(' ')[1];

Reply:
changed to fetch and find by id now

Reply:
and pass by param

how can i get id of user from your access token?